### PR TITLE
KOGITO-8145 Fix broken links in SWF Guide - section Security

### PR DESCRIPTION
**Pull request by zdrapela**
**JIRA:** https://issues.redhat.com/browse/KOGITO-8145

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributions doc](https://github.com/kiegroup/kogito-docs/blob/main/CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [x] The nav.adoc file has a link to this guide in the proper category
- [x] The index.adoc file has a card to this guide in the proper category, with a meaningful description

<details>
<summary>
How to setup automatic cherry-pick to another branch ?
</summary>

The cherry-pick action allows to setup automatic cherry-pick from `main` to a specific branch.

To allow it, you will need to add the corresponding label with pattern: `backport-{RELEASE_BRANCH}`.  
For example, if a backport to branch `1.26.x` is needed, then the label should be `backport-1.26.x`.

Once the PR is merged, the action will retrieve the commit and cherry-pick it to the desired branch.

*NOTE: You can still add label after the merge and it should still be cherry-picked.*
</details>

### CODE REVIEW

**Review by radtriste — APPROVED** [review]

**Review by wmedvede — APPROVED** [review]
Thank  @zdrapela for your help reviewing and fixing the guides, really much appreciated.!

File: serverlessworkflow/modules/ROOT/pages/security/authention-support-for-openapi-services.adoc [hunk 1]
```diff
@@ -45,7 +45,7 @@ The following shows the example of security scheme definitions:
 }
 ----
 
-If the OpenAPI specification file contains `securitySchemes` definitions, but not link:{open_api_spec_url}[Security Requirement Object] definitions, the generator can be configured to create the security requirement objects by default. In this case, for all operations without a security requirement, the default one is created. Note that the property value needs to match the name of a security scheme object definition, such as `http-basic-example` or `api-key-example` in the previous `securitySchemes` list.
+If the OpenAPI specification file contains `securitySchemes` definitions, but not link:{open_api_spec_url}#security-requirement-object[Security Requirement Object] definitions, the generator can be configured to create the security requirement objects by default. In this case, for all operations without a security requirement, the default one is created. Note that the property value needs to match the name of a security scheme object definition, such as `http-basic-example` or `api-key-example` in the previous `securitySchemes` list.
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/security/authention-support-for-openapi-services.adoc:48]:
```suggestion
If the OpenAPI specification file contains `securitySchemes` definitions, but not the link:{open_api_spec_url}#security-requirement-object[Security Requirement Object] definitions, the generator is configured to create the security requirement objects by default. In this case, for all the operations without a security requirement, the default one is created. Note that the property value must match the name of a security scheme object definition, such as `http-basic-example` or `api-key-example` in the previous `securitySchemes` list.
```

File: serverlessworkflow/modules/ROOT/pages/security/authention-support-for-openapi-services.adoc [hunk 2]
```diff
@@ -328,7 +328,7 @@ The following example shows `security-example.json` file, defining a `sayHelloOa
 }
 ----
 
-Unlike the `http basic`, `http bearer`, and `apiKey` security schemes, the OAuth 2.0 authentication relies on the link:https://quarkus.io/guides/security-openid-connect-client[Quarkus OpenId Connect (OIDC) and OAuth 2.0 Clients and Filters]. Therefore, you must add link:https://quarkus.io/guides/security-openid-connect-client#oidc-client-filter[Quarkus OIDC Client Filter Extension] to your project as shown in the following example:
+Unlike the `http basic`, `http bearer`, and `apiKey` security schemes, the OAuth 2.0 authentication relies on the link:https://quarkus.io/guides/security-openid-connect-client[Quarkus OpenId Connect (OIDC) and OAuth 2.0 Clients and Filters]. Therefore, you must add link:https://quarkus.io/guides/security-openid-connect-client-reference#oidc-client-filter[Quarkus OIDC Client Filter Extension] to your project as shown in the following example:
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/security/authention-support-for-openapi-services.adoc:331]:
```suggestion
Unlike the `http basic`, `http bearer`, and `apiKey` security schemes, the OAuth 2.0 authentication relies on the link:https://quarkus.io/guides/security-openid-connect-client[Quarkus OpenId Connect (OIDC) and OAuth 2.0 Clients and Filters]. Therefore, you must add the link:https://quarkus.io/guides/security-openid-connect-client-reference#oidc-client-filter[Quarkus OIDC Client Filter Extension] to your project as shown in the following example:
```

**Review by kaldesai — APPROVED** [review]
@zdrapela I have very minor suggetions. Otherwise, your content looks good to me. Thank you!

**Comment by zdrapela** [comment]
@kaldesai I agree with your suggestions. Thanks!

**Review by ricardozanini — APPROVED** [review]

**Pull request merged**